Index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="Style.css" />
+    <title>Portfolio</title>
+  </head>
+
+  <body>
+    <div class="mobile-container">
+      <nav class="topnav">
+        <a href="#home" class="active">My Logo</a>
+
+        <a class="icon">
+          <img src="images/btn.png" />
+        </a>
+      </nav>
+      <main>
+        <p class="headline">Hey There. I’m Yaser.</p>
```

Comment:
- [ ] The main title (e.g., "I'm Marianne ...") should be in a `h1` tag. You should change `p` tag  to `h1` tag here.